components/Cards/examples/resources/CardsAssets.xcassets/check-circle-blue.imageset/Contents.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Can we use [Material Icons](https://material.io/tools/icons/) instead? I believe for the catalog, we try to work with as many of the baseline styles/components as possible.  

If we need to have a circular background for the checkmark, can we look at adding a CALayer or something to the UIImageView instead?